tests/Tests/400__ods_dashboard/420__model_serving/420__model_serving.robot

```diff
@@ -1,48 +1,82 @@
 *** Settings ***
 Library           OperatingSystem
 Resource          ../../../Resources/Page/ODH/JupyterHub/HighAvailability.robot
-Suite Teardown    Teardown Model Serving
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHModelServing.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataConnections.resource
+Suite Setup       Model Serving Suite Setup
+# Suite Teardown    Teardown Model Serving
 
 
 *** Variables ***
 ${MODEL_MESH_NAMESPACE}=    mesh-test
 ${ODH_NAMESPACE}=    redhat-ods-applications
-${MS_REPO}=    https://github.com/opendatahub-io/modelmesh-serving
 ${EXPECTED_INFERENCE_OUTPUT}=    {"model_name":"example-onnx-mnist__isvc-b29c3d91f3","model_version":"1","outputs":[{"name":"Plus214_Output_0","datatype":"FP32","shape":[1,10],"data":[-8.233053,-7.7497034,-3.4236815,12.3630295,-12.079103,17.266596,-10.570976,0.7130762,3.321715,1.3621228]}]}
-
+${PRJ_TITLE}=    model-serving-project
+${PRJ_DESCRIPTION}=    project used for model serving tests
 
 *** Test Cases ***
-Validate Model Serving quickstart
-    [Documentation]    Test the quickstart of the model serving repo, Temporary until included in RHODS
-    [Tags]    ModelMesh_Serving
-    # TODO: Replace with http://robotframework.org/robotframework/latest/libraries/Process.html#Run%20Process
-    Run    git clone ${MS_REPO}
-    Run    cd modelmesh-serving/quickstart && ./quickstart.sh ${ODH_NAMESPACE} ${MODEL_MESH_NAMESPACE}
+# Validate Model Serving quickstart
+#     [Documentation]    Test the quickstart of the model serving repo, Temporary until included in RHODS
+#     [Tags]    ModelMesh_Serving
+#     # TODO: Replace with http://robotframework.org/robotframework/latest/libraries/Process.html#Run%20Process
+#     Run    git clone ${MS_REPO}
+#     Run    cd modelmesh-serving/quickstart && ./quickstart.sh ${ODH_NAMESPACE} ${MODEL_MESH_NAMESPACE}
 
 Verify Model Serving Installation
     [Documentation]    Verifies Model Serving resources
     [Tags]    ModelMesh_Serving
-    Run Keyword And Continue On Failure  Wait Until Keyword Succeeds  5 min  10 sec  Verify Openvino Deployment
+    # Run Keyword And Continue On Failure  Wait Until Keyword Succeeds  5 min  10 sec  Verify Openvino Deployment
     Run Keyword And Continue On Failure  Wait Until Keyword Succeeds  5 min  10 sec  Verify odh-model-controller Deployment
```

Comment:
where is it defined?

Reply:
Not sure to which one you are referring, but all "Verify xxx Deployment" should be among the keywords of the same file